src/annotator/integrations/index.ts

```diff
@@ -15,6 +16,10 @@ export function createIntegration(annotator: Annotator): Integration {
     return new PDFIntegration(annotator);
   }
 
+  if (isPdf2htmlEX()) {
```

Comment:
Ideally I would pass an integration through the [hypothesis config object](https://h.readthedocs.io/projects/client/en/latest/publishers/config.html#cmdoption-arg-contentReady), but this how its currently done for the existing PDF.js integration 🤷🏻‍♂️